### PR TITLE
add dashboard api

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

**Comment by Serabe** [comment]
I don't think having the API with the front project is a better approach than having the API within the actual server.

This is not an API building justice for this project, it is supposed to be a JSON version of the existing gRPC API. As such, the very best option would be to have it living side by side with the gRPC one. If not, it needs its own identity.

In any case, if someone wants to pursue this further, it needs a README on how to make it work.

**Comment by erizocosmico** [comment]
@Serabe there will be ad-hoc functionality such as gist saving and so on that should not be on `bblfsh/server`.

**Comment by Serabe** [comment]
Then, the second part applies.

**Comment by erizocosmico** [comment]
@Serabe done.

File: server/cmd/dashboard-server/main.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	var (
+		addr       string
+		bblfshAddr string
+	)
+
+	flag.StringVar(&addr, "addr", ":9999", "address in which the server will run")
+	flag.StringVar(&bblfshAddr, "bblfsh-addr", "0.0.0.0:9432", "address of the babelfish server")
+	flag.Parse()
+
+	r := gin.Default()
```
Comment (Serabe) [inline server/cmd/dashboard-server/main.go]: Is there any reason to have `r` defined here and not right before the routes?

Comment (erizocosmico) [inline server/cmd/dashboard-server/main.go]: Nope, moved stuff around and I left it there. Updated.

**Pull request merged**